### PR TITLE
E2E tests for configuration screen for different roles

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
E2E tests for configuration screen for different roles

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/2919227

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by prprabhu-ms** [comment]
This looks great. Let's have those snapshots so we can review the golden files as well.

File: packages/react-composites/tests/browser/call/app/QueryArgs.ts [hunk 1]
```diff
@@ -31,6 +33,7 @@ export function parseQueryArgs(): QueryArgs {
     userId: params.userId ?? '',
     groupId: params.groupId ?? '',
     token: params.token ?? '',
-    displayName: params.displayName ?? ''
+    displayName: params.displayName ?? '',
+    role: params.role as Role
```
Comment (alkwa-msft) [inline packages/react-composites/tests/browser/call/app/QueryArgs.ts:37]: If they don't pass in a role should we assume they are a presenter?

**Review by alkwa-msft — APPROVED** [review]

File: packages/react-composites/tests/browser/call/app/QueryArgs.ts (hunk 1 above)
Comment (mgamis-msft) [reply packages/react-composites/tests/browser/call/app/QueryArgs.ts:37]: I would say no. We don't want to pass the role Presenter to the composite for non-rooms calls.

**Review by carocao-msft — APPROVED** [review]

File: packages/react-composites/tests/browser/call/app/QueryArgs.ts (hunk 1 above)
Comment (prabhjot-msft) [reply packages/react-composites/tests/browser/call/app/QueryArgs.ts:37]: It is an optional prop, so I don't think we need to set default value

**Comment by dmceachernmsft** [comment]
image
is this update intentional?

**Comment by prabhjot-msft** [comment]
> image
> is this update intentional?

No, got updated automatically

**Review by prprabhu-ms — APPROVED** [review]

File: packages/react-composites/tests/browser/call/app/QueryArgs.ts (hunk 1 above)
Comment (JamesBurnside) [inline packages/react-composites/tests/browser/call/app/QueryArgs.ts:37]: nit: `as role | undefined;`

**Pull request merged**